vmdb/app/models/host.rb

```diff
@@ -1244,15 +1244,21 @@ def detect_discovered_os(ost)
   def detect_discovered_hypervisor(ost, ipaddr)
     log_header = 'MIQ(Host.detect_discovered_hypervisor)'
     find_method = :find_by_ipaddress
+
+    # TODO: break this apart, we probably want to only do the namelookup for hyperv, esx
+    hostname = Socket.getaddrinfo(ipaddr, nil)[0][2]
```

Comment:
I see the word timeout and what looks like a 5 second timeout, but I can't really grok what it's going: https://github.com/ruby/ruby/blob/ruby_2_0_0/lib/resolv.rb#L351.